gui/WiiLoadDialog.py

```diff
@@ -0,0 +1,110 @@
+import logging
+
+from PySide6 import QtCore
+from PySide6.QtCore import QSize, QStorageInfo, QDir, Qt, QEvent
+from PySide6.QtGui import QIcon, QGuiApplication, QFont
+from PySide6.QtWidgets import QDialog, QDialogButtonBox, QListWidgetItem, QMessageBox, QPlainTextEdit
+
+import gui_helpers
+import wiiload
+from gui import ui_WiiLoadDialog
+from utils import resource_path, file_size
+import serial
+import serial.tools.list_ports
```

Comment:
Some unused imports here
```suggestion
import logging

from PySide6.QtCore import Qt
from PySide6.QtGui import QIcon, QGuiApplication
from PySide6.QtWidgets import QDialog, QMessageBox

import gui_helpers
import wiiload
from gui import ui_WiiLoadDialog
from utils import resource_path
import serial
import serial.tools.list_ports
```